templates/base/footer_content.tmpl

```diff
@@ -23,7 +23,7 @@
 				{{end}}
 			</div>
 		</div>
-		<a href="{{AssetUrlPrefix}}/js/licenses.txt">{{.locale.Tr "licenses"}}</a>
+		<a href="{{AssetUrlPrefix}}/licenses.txt">{{.locale.Tr "licenses"}}</a>
```

Comment:
A longstanding question in my mind: this "license" file only contains frontend licenses? If not so, then it looks good to me.

If this file is frontend-only, does it need to show backend licenses?

Reply:
This file contains both frontend and backend licenses (extracted from `assets/go-licenses.json` above).